ynr/apps/elections/uk/management/commands/uk_create_elections_from_every_election.py

```diff
@@ -88,6 +93,23 @@ def import_approved_elections(
 
             election_dict.get_or_create_ballot(parent=parent)
 
+    def determine_voting_system(self, children):
+        """
+        Some ballots have a parent that is a top-level election (senedd, nia, parl).
+        We don't set voting_system for top-level elections in the EE API
+        so we need to determine it by looking at the voting systems of their children.
+        """
+        voting_systems = set()
+        for child in children:
+            if child["voting_system"]:
+                voting_systems.add(child["voting_system"]["slug"])
+        if len(voting_systems) > 1:
+            # No election_id one level up from a ballot should have multiple voting systems.
+            raise ValueError(
+                f"Multiple voting systems found for {children[0]['group']}"
```

Comment:
I know this is hard, but it might be worth suggesting an action here, or at least saying that no elections will be imported until this is fixed. Right now, if I saw this in Sentry I might not know how serious the error is.